### PR TITLE
Remove AQS based session ID dependency from Fireperf.

### PR DESCRIPTION
**Pull request by visumickey**

### CODE REVIEW

**Review by raymondlam — APPROVED** [review]
Are we just removing the AQS session id mapping for Fireperf events but still sending AQS session events and forwarding performance's collection status?

**Comment by visumickey** [comment]
> Are we just removing the AQS session id mapping for Fireperf events but still sending AQS session events and forwarding performance's collection status?

We are trying to make Fireperf not be impacted by AQS changes. Basically, for the multi process fix - we are taking an approach where Fireperf does not get impacted and then bring this change back in Fireperf. So, Fireperf will no longer get the sessionID from AQS until the multi process fix has landed.

File: firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java [hunk 1]
```diff
@@ -31,8 +31,6 @@
 import com.google.firebase.platforminfo.LibraryVersionComponent;
 import com.google.firebase.remoteconfig.RemoteConfigComponent;
 import com.google.firebase.sessions.FirebaseSessions;
```
Comment (mrober) [inline firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java:33]: Should we remove all references to `FirebaseSessions`?

Comment (visumickey) [reply firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java]: This is all cleaned up now after taking in the latest change of sessions-nine branch.

File: firebase-perf/CHANGELOG.md
Comment (mrober) [inline firebase-perf/CHANGELOG.md:1]: I think this file merged slightly wrong? You're change entry should go under unreleased and 20.5.0 is already out and the Kotlin block got removed.

**Review by mrober — APPROVED** [review]

File: firebase-perf/CHANGELOG.md
Comment (visumickey) [reply firebase-perf/CHANGELOG.md:1]: Just fixed the CHANGELOG to reflect the changes. Is the sessions-nine branch updated? I branched out of that branch.

**Pull request merged**